e2e/support/cypress_sample_instance_data.js

```diff
@@ -53,6 +53,9 @@ export const FIRST_COLLECTION_ID = _.findWhere(
   { name: "First collection" },
 ).id;
 
+/**
+ * @type number
+ */
```

Comment:
Before we end up converting all helpers to TS, some of the monkey-patching will be needed because TS complains in TS specs.